sdk/go/sdk/buildkite/notify.go

```diff
@@ -54,3 +52,59 @@ func (c StepNotify) toSchema() *schema.BlockStepNotify {
 
 	return notify
 }
+
+// Pipeline Notify
+type NotifyGithubCommitStatus struct {
+	Context *string
+}
+
+type PipelineNotify struct {
+	Email                *string                   `json:"email,omitempty"`
+	If                   *string                   `json:"if,omitempty"`
+	BasecampCampfire     *string                   `json:"basecamp_campfire,omitempty"`
+	Slack                *NotifySlack              `json:"slack,omitempty"`
+	Webhook              *string                   `json:"webhook,omitempty"`
+	PagerdutyChangeEvent *string                   `json:"pagerduty_change_event,omitempty"`
+	GithubCommitStatus   *NotifyGithubCommitStatus `json:"github_commit_status,omitempty"`
```

Comment:
Nit, but some folks tend to get irritated when that `H` isn't capped 😂 
```suggestion
	GitHubCommitStatus   *NotifyGithubCommitStatus `json:"github_commit_status,omitempty"`
```

Reply:
☝🏻 We should change this across the board by the way, in all languages.